engine/src/main/java/org/hibernate/search/query/dsl/impl/FacetRange.java

```diff
@@ -87,6 +87,20 @@ private String buildRangeString() {
 	}
 
 	@Override
+	public int hashCode() {
```

Comment:
Ok, cool. Just let me know then when you're done. Any other loose ends from your PoV related to faceting (apart from the overall re-design we discussed ;)?